deb-package/pkg/lib/systemd/system/dvoting.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=D-Voting Node Service
+After=network.target
+Requires=unikernel.service
+
+[Service]
+User=dvoting
+
+ExecStartPre=/bin/bash -c "rm -f /var/opt/dedis/dvoting/data/dela/daemon.sock"
```

Comment:
this seems hacky, but I guess we can live with it for now 🤷 
Just for future reference, the cleaner way would be to rely on `/run` or `/var/run` .